apps/metadata-editor/src/app/records/search-records/search-records-list.component.spec.ts

```diff
@@ -1,30 +1,85 @@
 import { ComponentFixture, TestBed } from '@angular/core/testing'
-import { SearchFacade } from '@geonetwork-ui/feature/search'
+import { SearchFacade, SearchService } from '@geonetwork-ui/feature/search'
 import { SearchRecordsComponent } from './search-records-list.component'
-import { Component, importProvidersFrom, Input } from '@angular/core'
+import {
+  Component,
+  EventEmitter,
+  importProvidersFrom,
+  Input,
+  Output,
+} from '@angular/core'
 import { TranslateModule } from '@ngx-translate/core'
 import { BehaviorSubject } from 'rxjs'
-import { RecordsListComponent } from '../records-list.component'
+import { CatalogRecord } from '@geonetwork-ui/common/domain/model/record'
+import { CommonModule } from '@angular/common'
+import { MatIconModule } from '@angular/material/icon'
+import { UiInputsModule } from '@geonetwork-ui/ui/inputs'
+import { By } from '@angular/platform-browser'
+import { DATASET_RECORDS } from '@geonetwork-ui/common/fixtures'
+import { Router } from '@angular/router'
+
+const results = [{ md: true }]
+const currentPage = 5
+const totalPages = 25
+
+@Component({
+  // eslint-disable-next-line
```

Comment:
Not even specifying which rule to disable?

Reply:
we've always been doing like that in tests when defining mock components because otherwise eslint complains about the component prefix; but I agree that specifying the rule to disable would make it more explicit